src/modules/imageresizer/ui/Views/InputPage.xaml.cs

```diff
@@ -3,12 +3,54 @@
 // See the LICENSE file in the project root for more information.  Code forked from Brice Lambson's https://github.com/bricelam/ImageResizer/
 
 using System.Windows.Controls;
+using System.Windows.Input;
+using ImageResizer.ViewModels;
+using ModernWpf.Controls;
+using static ImageResizer.ViewModels.InputViewModel;
 
 namespace ImageResizer.Views
 {
     public partial class InputPage : UserControl
     {
         public InputPage()
             => InitializeComponent();
+
+        private void Button_KeyDown(object sender, KeyEventArgs e)
+        {
```

Comment:
Need some comments here to explain that we are doing this so that the values being inserted are really taken into account when the user presses Enter while editing one of the text boxes.